config/migrations/2024/20240910102800-legislation-2025/20240923130916-tmp-ocmw-bestuursorganen.sparql

```diff
@@ -28,6 +29,7 @@ INSERT {
           mu:uuid ?uuidInTijd;
           ext:origineleBestuursorgaan ?bestuursorgaanInTijd;
           lmb:heeftBestuursperiode ?bestuursperiode;
+          mandaat:bindingStart ?startDate;
```

Comment:
This already fixes the issue, but this will only be run when the database is thrown away again, thats why I added the other one as a local migration